packages/blade/src/components/Input/BaseInput/baseInputStyles.ts

```diff
@@ -46,60 +54,78 @@ export const getInputBackgroundAndBorderStyles = ({
   | 'isDropdownTrigger'
 >): CSSObject => {
   // normal state
-  let backgroundColor = theme.colors.interactive.background.gray.default;
-  let borderBottomColor = theme.colors.interactive.background.gray.highlighted;
+  let backgroundColor = getIn(theme.colors, baseInputBackgroundColor.default);
+  let borderColor = getIn(theme.colors, baseInputBorderColor.default);
+  let borderWidth: Theme['border']['width'][keyof Theme['border']['width']] = getIn(
```

Comment:
🤔 given that GetIn is typesafe, do we need to explicitly add this type? `Theme['border']['width'][keyof Theme['border']['width']]`

Reply:
If for some reason we do, let's use `Border['width'][keyof Border['width']]` bit smaller